config/routes.rb

```diff
@@ -1,5 +1,12 @@
 Rails.application.routes.draw do
+  get 'posts/create'
+  get 'posts/new'
+  get 'posts/edit'
+  get 'posts/show'
+  get 'posts/update'
+  get 'posts/destroy'
```

Comment:
Since you already have routes for the posts below:

```ruby
  resources :forums do
    resources :posts, shallow: true, except: [:index]
    ...
```

These ones would only interfere with your app. It also wouldn't make sense to use `get` for the `update` and `destroy` actions because those modify data, and `get` is only for reading data.